pallets/external-validator-slashes/src/lib.rs

```diff
@@ -122,6 +126,11 @@ pub mod pallet {
 
         /// Invulnerable provider, used to get the invulnerables to know when not to slash
         type InvulnerablesProvider: InvulnerablesProvider<Self::ValidatorId>;
+        type ValidateMessage: ValidateMessage;
+        type OutboundQueue: SendMessage<
```

Comment:
I am wondering whether it makes sense to add our own `SendMessage` trait and implement it for things that implement their `SendMessage` (essentially being a wrapper here). This suggestion comes from the fact that we are implement their `sendMessage` in our mocks which does not make sense since we have our own custom validation